test/post.test.js

```diff
@@ -307,6 +307,9 @@ describe('Post data class tests', () => {
       expect(videoPost.title).toBe('DPS909 OSD600 Week 03 - Fixing a Bug in the Azure JS SDK');
       expect(videoPost.url).toBe('https://www.youtube.com/watch?v=mNuHA7vH6Wc');
       expect(videoPost.type).toBe('video');
+      expect(videoPost.html).toBe(
```

Comment:
OK, but let's update the test title to indicate that you're also testing for proper content handling:

```js
test('Post.createFromArticle() should properly create Post for YouTube, with linkified content', async () => {...});
```